### DISCUSSION TITLE
Remove top image from blog post

### DISCUSSION THREAD

**Discussion by Orilcious** [thuliteio/doks#1094] [Q&A]
Hello,

is there a way to use an image in the preview of a blogpost without having it at the top of the post as well?

I found https://github.com/h-enk/doks/discussions/999 where it was not yet possible.
Has this been added by now?

Kind regards,
Ori

**Comment by h-enk** [comment]
Not yet. I will add this alongside some image size normalizations. for blog posts.

**Reply by Orilcious** [reply]
Ok I will keep my posts without images until then. Thanks for your work!
